test/reactors/V3DutchOrderReactor.t.sol

```diff
@@ -0,0 +1,1005 @@
+        CosignerData memory cosignerData = CosignerData({
+            decayStartBlock: block.number,
+            exclusiveFiller: address(1),
+            exclusivityOverrideBps: 0,
+            inputAmount: 0,
+            outputAmounts: ArrayBuilder.fill(1, 0)
+        });
+
+        V3DutchOrder memory order = V3DutchOrder({
+            info: OrderInfoBuilder.init(address(reactor)).withSwapper(swapper),
+            cosigner: vm.addr(cosignerPrivateKey),
+            baseInput: V3DutchInput(tokenIn, inputAmount, CurveBuilder.emptyCurve(), inputAmount),
+            baseOutputs: OutputsBuilder.singleV3Dutch(
+                address(tokenOut), 1 ether, CurveBuilder.singlePointCurve(1, 0.1 ether), swapper
+            ),
+            cosignerData: cosignerData,
+            cosignature: bytes("")
+        });
+        order.cosignature = cosignOrder(order.hash(), cosignerData);
+        SignedOrder memory signedOrder =
+            SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
+        vm.expectRevert(ExclusivityLib.NoExclusiveOverride.selector);
+        fillContract.execute(signedOrder);
+    }
+
+    function testV3StrictExclusivityValidCaller() public {
+        uint256 inputAmount = 1 ether;
+        uint256 outputAmount = 1 ether;
+        tokenIn.mint(swapper, inputAmount);
+        tokenOut.mint(address(fillContract), inputAmount);
+        tokenIn.forceApprove(swapper, address(permit2), type(uint256).max);
+        CosignerData memory cosignerData = CosignerData({
+            decayStartBlock: block.number,
+            exclusiveFiller: address(fillContract),
+            exclusivityOverrideBps: 0,
+            inputAmount: 0,
+            outputAmounts: ArrayBuilder.fill(1, 0)
+        });
+
+        V3DutchOrder memory order = V3DutchOrder({
+            info: OrderInfoBuilder.init(address(reactor)).withSwapper(swapper),
+            cosigner: vm.addr(cosignerPrivateKey),
+            baseInput: V3DutchInput(tokenIn, inputAmount, CurveBuilder.emptyCurve(), inputAmount),
+            baseOutputs: OutputsBuilder.singleV3Dutch(
+                address(tokenOut), 1 ether, CurveBuilder.singlePointCurve(1, 0.1 ether), swapper
+            ),
+            cosignerData: cosignerData,
+            cosignature: bytes("")
+        });
+        order.cosignature = cosignOrder(order.hash(), cosignerData);
+        SignedOrder memory signedOrder =
+            SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
+        vm.prank(address(1));
+
+        _snapStart("V3-ExclusiveFiller");
+        fillContract.execute(signedOrder);
+        snapEnd();
+
+        assertEq(tokenIn.balanceOf(swapper), 0);
+        assertEq(tokenOut.balanceOf(swapper), outputAmount);
+        assertEq(tokenIn.balanceOf(address(fillContract)), inputAmount);
+    }
+
+    function testV3ExclusiveOverrideInvalidCallerCosignedAmountOutput() public {
```

Comment:
this is not invalidCaller since it succeeds? Maybe like "testV3AppliesExclusiveOverride..."